bridge/models.py

```diff
@@ -79,9 +79,43 @@ class Trump(Enum):
 
 class Hand:
     def __init__(self, cards=None):
-        self.cards = cards
+        self.cards = {};
```

Comment:
Could use python built-in lib: defaultdict https://docs.python.org/2/library/collections.html#collections.defaultdict
```python
self.cards = defaultdict(list)
```

So we don't need to setup default value